atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/NonOverlappingReadsMonotonicChecker.java

```diff
@@ -87,10 +87,14 @@ public boolean valid() {
 
         private void validateTimestampHigherThanReadsCompletedBeforeInvoke(InvokeEvent invoke, OkEvent event) {
             OkEvent lastAcknowledgedRead = lastAcknowledgedReadBefore(invoke.time());
-            if (lastAcknowledgedRead != null && lastAcknowledgedRead.value() >= event.value()) {
-                errors.add(lastAcknowledgedRead);
-                errors.add(invoke);
-                errors.add(event);
+            if (lastAcknowledgedRead != null) {
+                Long timestamp = Long.parseLong(event.value());
+                Long lastAcknowledgedTimestamp = Long.parseLong(lastAcknowledgedRead.value());
```

Comment:
as above - because we're only doing comparisons there's no need to box the output of `Long.parseLong()`